UnrealPlugin/README.md

```diff
@@ -10,337 +11,38 @@ It was tested with Unreal Engine 4.26.2, but should work with other engines as w
-navigate to Maps&Modes on the left side. Scroll to the bottom, and then you can set the option "Game Instance Class" to your new game instance class directly, and avoid typos.
-
-###### In DefaultEngine.ini
-Or you can update DefaultEngine.ini file and add this:
-```ini
-[/Script/EngineSettings.GameMapsSettings]
-GameInstanceClass=/Script/[game name].MyGameInstance
-```
-
-## Include Pre-requisites for Windows Dedicated Server
-There are two ways to include the app-local prerequisites - either through the Unreal Engine editor or by editing DefaultGame.ini.
-
-### In the Unreal Editor
-In the editor go to Edit -> Project Settings. In the opened window navigate to Packaging on the left side. Scroll to the bottom of the list, and tick "Include app-local prerequisites".
-
-### In DefaultGame.ini
-Or you can update DefaultGame.ini to show the following:
-```ini
-[/Script/UnrealEd.ProjectPackagingSettings]
-IncludeAppLocalPrerequisites=True
-```
-
-If the category already exists in your DefaultGame.ini, then just add the second line to it. This ensures that all app local dependencies ship with the game as well.
-
-If you are using Continuous Integration (CI), then you could add it to your setup to only turn this flag on when building a dedicated server, so the additional dlls only get added if it is a dedicated server build.
-
-## Packaging the game server
-
-Make sure that you have just built your project in development editor in Visual Studio *before* doing this next step, because sometimes building fails when opening the project directly through the .uproject file.
+## Requirements
 
-Right click on the .uproject file in your file editor and select "Generate Visual Studio Project Files". 
+* Download Visual Studio (the [community version is free](https://visualstudio.microsoft.com/vs/community/))
+	- Requires Workloads: .NET desktop development, and Desktop development with C++
+* Download Unreal Engine Source Build by following [these instructions](https://docs.unrealengine.com/4.26/ProgrammingAndScripting/ProgrammingWithCPP/DownloadingSourceCode/) from the Unreal Engine website. This was tested on [4.26.2](https://github.com/EpicGames/UnrealEngine/releases/tag/4.26.2-release) (Link requires permissions)
+* Download the [Unreal PlayFabGSDK Plugin folder](https://github.com/PlayFab/gsdk/tree/master/UnrealPlugin)
+* [Optional] Download the [LocalMultiplayerAgent](https://github.com/PlayFab/MpsAgent/releases)
+	* [Optional] Alternately, download the [LocalMultiplayerAgent source](https://github.com/PlayFab/MpsAgent/tree/main/LocalMultiplayerAgent)
+* [Optional] [PlayFab Marketplace plugin](https://www.unrealengine.com/marketplace/product/playfab-sdk) or the [source version on GitHub](https://github.com/PlayFab/UnrealMarketplacePlugin/tree/master/4.26/PlayFabPlugin/PlayFab) (Not required for GSDK, but required for most other PlayFab services, including login)
```

Comment:
This plugin is not required for GSDK but it's required for many PlayFab services, including login.